pkg/cmd/cli/cmd/tag.go

```diff
@@ -133,7 +133,7 @@ func parseStreamName(defaultNamespace, name string) (string, string, error) {
 
 func determineSourceKind(f *clientcmd.Factory, input string) string {
 	mapper, _ := f.Object()
-	gvks, err := mapper.KindsFor(schema.GroupVersionResource{Group: imageapi.GroupName, Resource: input})
+	gvks, err := mapper.KindsFor(schema.GroupVersionResource{Group: imageapi.LegacyGroupName, Resource: input})
```

Comment:
The upstream change is here introducing ResourcesShortcutStatic: https://github.com/kubernetes/kubernetes/commit/ad688d77619955f793b6989ba44f0ecca602ab46. Was merged shortly after 1.5.

But, similar lists existed before: https://github.com/kubernetes/kubernetes/commit/94d683e89baba6f7165f4e872ea069b9c371ae63